src/Thujohn/Twitter/Traits/AccountActivityTrait.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Thujohn\Twitter\Traits;
+
+use BadMethodCallException;
+
+trait AccountActivityTrait
+{
+	/**
+	 * Creates HMAC SHA-256 hash from incomming crc_token and consumer secret.
+	 * This base64 encoded hash needs to be returned by the application when Twitter calls the webhook.
+	 *
+	 * @param  mixed $crcToken
+	 *
+	 * @return void
+	 */
+	public function crcHash($crcToken)
+	{
+		$hash = hash_hmac('sha256', $crcToken, $this->tconfig['CONSUMER_SECRET'], true);
+
+		return 'sha256='.base64_encode($hash);
+	}
+	
+	/**
+	 * Registers a webhook $url for all event types in the given environment.
+	 *
+	 * @param  mixed $env
+	 * @param  mixed $url
+	 *
+	 * @return void
+	 */
+	public function setWebhook($env, $url)
+	{
+		return $this->post("account_activity/all/{$env}/webhooks", ['url' => $url]);
+	}
+
+	/**
+	 * Returns webhook URLs for the given environment (or all environments if none provided), and their statuses for the authenticating app. 
+	 *
+	 * @param  mixed $env
+	 *
+	 * @return void
+	 */
+	public function getWebhooks($env = null)
+	{
+		return $this->get("account_activity/all/".($env ? $env."/" : ""). "webhooks");
+	}
+
+	/**
+	 * Triggers the challenge response check (CRC) for the given enviroments webhook for all activites.
+	 * If the check is successful, returns 204 and reenables the webhook by setting its status to valid.
+	 *
+	 * @param  mixed $env
+	 * @param  mixed $webhookId
+	 *
+	 * @return void
+	 */
+	public function updateWebhooks($env, $webhookId)
+	{
+		$this->query("account_activity/all/{$env}/webhooks/{$webhookId}", "PUT");
+
+		return $this->response['code'] == 204 ? true : false;
```

Comment:
This can just be (condition output will be a bool) (and below)
```suggestion
		return $this->response['code'] === 204;
```